### ISSUE TITLE
Release for inclusion in LaMachine v2

### ISSUE THREAD

**Issue by proycon** [ErwinKomen/FoliaEntity#8]
Hi Erwin, 

I'm working on a new LaMachine v2 and would like to include foliaentity again, so everybody can use it  (see proycon/LaMachine#57 ). The current solution in LaMachine v1 was a bit patchy, would it be possible to make use of the release mechanism here on github for foliaentity (as opposed to the dropbox link) ?  Since you distribute a binary release just uploading that archive when doing the github release should be a good method. 

Ideally, I'd also like to be able to compile it from source in LaMachine v2, but I don't know much about C# and Mono.

**Comment by ErwinKomen** [comment]
Let me check if I understand this correctly. Are you saying I should include the _binary_ here in github?

I agree that source-compiling from C# would be the better solution.
However... that's not a simple task. 
Easiest solution is (if you're working from Linux):
1. Install windows in a VM
1. Install the free version of Microsoft Visual Studio  (vs2015 or even vs2017)
1. Load the sources into a project for VS
1. Compile them over there

(And if you have Windows somewhere, then stap [1] is not needed obviously.)

**Comment by proycon** [comment]
> Let me check if I understand this correctly. Are you saying I should include the binary here in github?

Yes, not in git obviously, but as an attached archive (zip) when doing a release on github.

> I agree that source-compiling from C# would be the better solution.
> However... that's not a simple task.
> Easiest solution is (if you're working from Linux):
> 
>     Install windows in a VM
>     Install the free version of Microsoft Visual Studio (vs2015 or even vs2017)
>     Load the sources into a project for VS
>     Compile them over there

Okay, you lost me at step 1 already, that is indeed not an option then :) I had hoped it could be done with only mono but I don't know much about it.

**Comment by ErwinKomen** [comment]
1. Compiled attachment: I'm going to look into that today
1. C# project setup and compilation: it seems that it may be possible on Linux by using (a) [Visual Studio Code](https://code.visualstudio.com/docs/languages/csharp) and/or (b) downloading the [.NET core SDK](https://www.microsoft.com/net/download/linux). I have no experience with this, but using the MS tools seems like a promising way to get the code compiled properly.

**Comment by ErwinKomen** [comment]
I've now added the 'released' 'exe' file to the repository.
It's on this path:
`FoliaEntity` / `FoliaEntity` / `bin` / `Release`

**Comment by proycon** [comment]
Thanks, though that's not exactly what I meant still :) I meant here: https://github.com/ErwinKomen/FoliaEntity/releases/ 

See also https://help.github.com/articles/creating-releases/

**Comment by proycon** [comment]
That .NET core SDK option might indeed be a viable route to compile from source, we'd have to investigate that (but not a very big priority)
